### DISCUSSION TITLE
How do I change the configuration after installation?

### DISCUSSION THREAD

**Discussion by SparkMike77** [tsightler/ring-mqtt#306] [Support Q&A] [answered]
Installed in HomeAssistant, via the app store.

Tried searching through the documentation, sorry if I missed it.

Specifically I'd like to enable the camera functions, but the webUI functions are just the login prompt for my ring account or the notification that it's already connected to Ring.

**Comment by tsightler** [comment, marked as answer]
It's an addon so the configuration is directly in the HA UI.  Only the initial authentication is done via the addons built-in web app.  Exactly how to do this varies based on whether you are using a Desktop browser or a mobile browser, but basically, when you click on the addon there are 4 tabs on the top (Info, Documentation, Configuration, Log), or, if on mobile browser, 4 icons at the bottom.  Either way, it's the same, the configuration tab.

**Reply by SparkMike77** [reply]
Sir, you are a scholar and a gentleman, thank you.  This is my very first community add-on and it's almost comical how much better it is than the older "official HA" one since Ring started messing about.
